packages/platforms/browser/lib/on-settle/load-event-end-settler.ts

```diff
@@ -52,8 +52,10 @@ class LoadEventEndSettler extends Settler {
 
     if (isPerformanceNavigationTiming(entry)) {
       settledTime = entry.loadEventEnd
-    } else {
+    } else if (performance.timing) {
       settledTime = performance.timing.loadEventEnd - performance.timing.navigationStart
+    } else {
+      settledTime = 0
```

Comment:
I added this so that I was able to unit test AngularRoutingProvider was setting the span URL after onSettle. This is perhaps not the way we want to handle this but thought I'd leave it here for discussion as we don't elsewhere seem to be testing the actual exported onSettle (we could add similar coverage to vue-router and react-router).

Without this allowing onSettle to trigger in a test fails because window.performance.timing is not defined in jsdom

Reply:
It sounds like an integration test for onSettle might be a good idea - we do mock quite a lot of implementation.

totally out of scope for this PR though :)

Reply:
Do we need to add anything at all here? The fallback to use `now` would trigger if `settledTime` is `undefined` so I think we only need the `else if (performance.timing)` bit